_posts/2020-08-05-hive-acid-original-files.md

```diff
@@ -42,32 +54,41 @@ columns and have differences in the schema as follows:
 
 
 
-Before Presto 340, Presto used to fail the query if it reads from a Hive transactional table having original files.
+Before Presto 340, Presto used to fail the query if it reads from a Hive
+transactional table having original files.
 
 # Update and delete support on original files
 
-Hive achieves updates/deletes on a row in original files by synthetically generating ACID columns for original files. Presto follows the same mechanism of generating ACID columns
-synthetically as discussed later.
+Hive achieves updates/deletes on a row in original files by synthetically
+generating ACID columns for original files. Presto follows the same mechanism of
+generating ACID columns synthetically as discussed later.
 
 ## ACID column generation on original files
 
-Files in Hive ACID format have 5 ACID columns, but we need only 3 columns i.e. `originalTransactionId`, `bucketId` and `rowId` to uniquely identify a row. In this section, we will
-see how these 3 columns are synthetically generated for original files.
+Files in Hive ACID format have 5 ACID columns, but we need only 3 columns i.e.
+`originalTransactionId`, `bucketId` and `rowId` to uniquely identify a row. In
+this section, we will see how these 3 columns are synthetically generated for
+original files.
 
 ### Original transaction ID
 
-An original transaction ID is the write ID when a record is first created. For original files, the original transaction ID is always 0.
+An original transaction ID is the write ID when a record is first created. For
+original files, the original transaction ID is always 0.
 
 ### Bucket ID
 
-Bucket ID is retrieved from the original file name. For the original file `0000ABC_DEF` or `0000ABC_DEF_copy_G`, the bucket ID will be `ABC`.
+Bucket ID is retrieved from the original file name. For the original file
+`0000ABC_DEF` or `0000ABC_DEF_copy_G`, the bucket ID will be `ABC`.
 
 ### Row ID
 
-To calculate the row ID, the total row count of all the original files, which come before the current one in lexicographical order, is calculated.
-Then, the row ID is equal to the sum of the value calculated and local row ID in the current original file.
+To calculate the row ID, the total row count of all the original files, which
+come before the current one in lexicographical order, is calculated.
+Then, the row ID is equal to the sum of the value calculated and local row ID in
+the current original file.
 
-Here is an example to calculate the global Row ID of the 3rd row of an original File `000000_0_copy_2`.
+Here is an example to calculate the global Row ID of the 3rd row of an original
+File `000000_0_copy_2`.
```

Comment:
@mosabua: Skipped L99 for 80 characters hard line wrap, since markdown was rendering it as it is.

Reply:
yes .. good catch..